docs/src/main/paradox/docs/delta/api/resolvers-api.md

```diff
@@ -250,15 +250,18 @@ if the `Accept` header is set to `text/html`, a redirection to the fusion repres
 ## Fetch original payload
 
 ```
-GET /v1/resolvers/{org_label}/{project_label}/{resolver_id}/source?rev={rev}&tag={tag}
+GET /v1/resolvers/{org_label}/{project_label}/{resolver_id}/source?rev={rev}&tag={tag}&annotate={annotate}
```

Comment:
This should go in `resources-api.md`
Also this is not mandatory for this PR but we try to include a snapshot in the description of the PR to make sure the rendering is ok